pkg/init/usermode-helper.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+int main(int argc, char *argv[])
+{
+	if (!strcmp(argv[0], "/sbin/mdev")) {
+		/* busybox uses /sbin/mdev for early uevent bootstrapping */
+		execv(argv[0], argv);
+	} else if (!strcmp(argv[0], "/sbin/modprobe")) {
+		/* allow modprobe */
+		execv(argv[0], argv);
+	} else {
```

Comment:
we probably want to add `/sbin/reboot` and `/sbin/poweroff`as this is used by `orderly_reboot() ` and `orderly_poweroff()` in `kernel/reboot.c` and on Hyper-V this can be triggered from the host (see ` drivers/hv/hv_util.c`